### PR TITLE
Fixed transparent area behind file manager.

### PR DESCRIPTION
**Pull request by trungnt2910**
I've fixed issue #2 
Also, I've fixed `PListItem.h` by adding a missing <cstring> header, so that the file can build.

### CODE REVIEW

File: source/WindowPeek.cpp [hunk 1]
```diff
@@ -357,8 +357,8 @@ WindowPeek::WindowPeek( BRect R, char* name , Setup* s, Language *w)
 
   // ok
   
-  alles->SetViewColor( mainMenu->ViewColor() );
-  filePane->SetViewColor( mainMenu->ViewColor() );
+  alles->SetViewColor(216,216,216);
```
Comment (humdingerb) [inline source/WindowPeek.cpp]: Instead of hard-coding the colour, use `ui_color(B_PANEL_BACKGROUND_COLOR)`

**Comment by humdingerb** [comment]
Thanks!

File: source/PListItem.h [hunk 2]
```diff
@@ -5,6 +5,7 @@
 #include <View.h>
 #include <Entry.h>
 #include <stdlib.h>
+#include <cstring>
```
Comment (korli) [inline source/PListItem.h]: PListItem.h doesn't need this header.

Comment (trungnt2910) [reply source/PListItem.h]: Yes it does, for strcmp.

Comment (korli) [reply source/PListItem.h]: Do you see any strcmp in https://github.com/HaikuArchives/Peek/blob/master/source/PListItem.h?

Comment (trungnt2910) [reply source/PListItem.h]: complaints
Here are `make`'s complaints when I comment out the `<cstring>` header.

Comment (korli) [reply source/PListItem.h]: These complaints are in PListItem.cpp

Comment (trungnt2910) [reply source/PListItem.h]: Oh, I see it now. I've moved the header include to the `PListItem.cpp` instead. 
The changes are added to my other pull request, #20

**Comment by owenca** [comment]
@trungnt2910 See my comments [here](https://github.com/HaikuArchives/Peek/pull/18#issuecomment-573973242).

**Pull request merged**